app/src/main/java/co/devskills/springbootboilerplate/advice/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,24 @@
+package co.devskills.springbootboilerplate.advice;
+
+import java.util.Date;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import co.devskills.springbootboilerplate.exception.ApplicationException;
+import co.devskills.springbootboilerplate.exception.ErrorDetails;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+	@ExceptionHandler(ApplicationException.class)
+	public ResponseEntity<ErrorDetails> applicationExceptionHandler(ApplicationException applicationException,
```

Comment:
Create a  global exception handler to throw any meaningful exception in the application to the end user